# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/capi.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- linux-2.6.8.1/include/linux/capi.h~	2004-08-14 12:55:33.000000000 +0200
+++ linux-2.6.8.1/include/linux/capi.h	2004-10-14 17:18:13.552303472 +0200
@@ -14,6 +14,7 @@
 
 #include <asm/types.h>
 #include <linux/ioctl.h>
+#include <linux/compiler.h>
 #ifndef __KERNEL__
 #include <linux/kernelcapi.h>
 #endif
